i18n/states/TZ.php

```diff
@@ -0,0 +1,37 @@
+<?php
+/**
+ * TZ states
+ *
+ * @author      Lev @ Extreme Web Technologies Ltd
+ * @category    i18n
+ * @package     WooCommerce/i18n
+ * @version     2.0.0
+ */
+global $states;
+
+$states['TZ'] = array(
+	'AR'  => __( 'Arusha', 'woocommerce' ),
```

Comment:
The list of states should follow ISO 3166-2. Please see: https://en.wikipedia.org/wiki/ISO_3166-2:TZ

Reply:
Thank you, done - updated all of them based on the Wikipedia link.